CHANGELOG.rst

```diff
@@ -4,10 +4,11 @@ CHANGELOG
 This document describes changes between each past release.
 
 
-0.6.0 (unreleased)
+0.6.0 (2015-10-12)
 ==================
 
-- Nothing changed yet.
+- Supports requests-hawk 0.2.0 drawbacks. (#17)
```

Comment:
Ah, I get it now. I would say "Circumvent a limitation present in requests-hawk 0.2.0"